monitor_check.py

```diff
@@ -26,6 +28,11 @@
 TRACKER = 1890881  # PYTHON3.10
 LOGLEVEL = logging.WARNING
 
+RAWHIDEVER = "rawhide"
+sacks = {}  # a global registry of dnf sacks
```

Comment:
Since this code only uses one sack, you might avoid this dict entirely and just do:

```python
RAWHIDE_SACK = rawhide_sack()
```

See for example https://github.com/hroncok/compare_deps/blob/main/compare_deps.py